tox.ini

```diff
@@ -15,11 +15,13 @@ commands=flake8 dag
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}
-deps =
-    -r{toxinidir}/requirements_dev.txt
+
+deps=
+    .
```

Comment:
The `.` is to make sure that wheel-only dependencies get installed before `setup.py` starts looking for them